From e693f2d7f1977ca227589154a5cd8c18d8ce44b7 Mon Sep 17 00:00:00 2001
From: Ahmad Samir <a.samirh78@gmail.com>
Date: Wed, 1 Sep 2021 11:38:29 +0200
Subject: [PATCH] The default navigation method should be TabbedNavigation

TabbedNavigation is when we have a MainWindow, i.e. the typical use case;
whereas NoNavigation is when using Konsole Part. The code in Part calls
setNavigationMethod(NoNavigation), so things should work as before.

I made a wrong assumption that TabbedNavigation was already the default.

CCBUG: 432077
---
 src/ViewManager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ViewManager.cpp b/src/ViewManager.cpp
index 2ba77e9f0..3ffbe6ba0 100644
--- a/src/ViewManager.cpp
+++ b/src/ViewManager.cpp
@@ -52,7 +52,7 @@ ViewManager::ViewManager(QObject *parent, KActionCollection *collection) :
     _pluggedController(nullptr),
     _sessionMap(QHash<TerminalDisplay *, Session *>()),
     _actionCollection(collection),
-    _navigationMethod(NoNavigation),
+    _navigationMethod(TabbedNavigation),
     _navigationVisibility(NavigationNotSet),
     _managerId(0),
     _terminalDisplayHistoryIndex(-1)
-- 
GitLab

